install.js

```diff
@@ -0,0 +1,80 @@
+var os = require("os");
+var path = require("path");
+var ProgressBar = require("progress");
+var get = require("simple-get");
+var tar = require("tar");
+var ffmpegPath = require(".");
+var pkg = require("./package");
+
+function downloadExtractFile(url, destinationPath, progressCallback) {
+  let fulfill, reject;
+  let downloadedBytes = 0;
+  let totalBytes = 0;
+
+  const promise = new Promise((x, y) => {
+    fulfill = x;
+    reject = y;
+  });
+
+  get(url, function(err, response) {
+    if (err || response.statusCode !== 200) {
+      const error = new Error(`Download failed. URL: ${url}`);
+      reject(error);
```

Comment:
To retain helpful debugging info, add the original `err` to `error`, or reject with `err` itself.